### DISCUSSION TITLE
I do not understand the guide can someone send me a video guide? [Question]

### DISCUSSION THREAD

**Discussion by baruchtasma1** [getmango/Mango#178] [Q&A]
Thanks for answering

**Comment by hkalexling** [comment]
Hi, it would be best if you could let us know the exact problem you are having with the guide. README is the standard documentation for open source projects, and I don't have the time to maintain a video and update it every time there's a new release. I will be happy to help if you have a concrete question.

**Comment by ghost** [comment]
Installed it as docker on unRAID. Where would i find STDOUT file password?

**Reply by hkalexling** [reply]
I don't use unRaid, and the unRaid template is not maintained by me, but I think there should be an option in unRaid to show the container log. You can find your default password in the log.

For future reference, please don't comment on an existing discussion unless you are answering it. If you have a new question, please open a new issue/discussion thread.
